lib/tirexs/query.ex

```diff
@@ -11,11 +11,15 @@ defmodule Tirexs.Query do
   end
 
   @doc false
-  def _query(options, query_opts\\[]) do
-    if is_list(options) do
-      query_opts = Enum.fetch!(options, 0)
-      options = extract_do(options, 1)
-    end
+  def _query(options, query_opts \\ [])
+
+  @doc false
+  def _query(options, _query_opts) when is_list(options) do
+    [query: extract(extract_do(options, 1)) ++ Enum.fetch!(options, 0)]
```

Comment:
yes, i think about readable code. If we use variables, it's more easy understand what happens in this line. But when i see big lines, it's so hard. Tirexs code so open for new developers, and elixir new language, we need make code easy as possible. It's Tirexs idea.